enhancements/cvo/cluster-profiles.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Cluster Profiles
+authors:
+  - "@csrwng"
+reviewers:
+  - "@abhinavdahiya"
+  - "@crawford"
+  - "@sdodson"
+  - "@derekwaynecarr"
+  - "@smarterclayton"
+approvers:
+  - "@derekwaynecarr"
+  - "@smarterclayton"
+creation-date: 2020-02-04
+last-updated: 2019-02-04
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# cluster-profiles
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+Cluster profiles are a way to support different deployment models for OpenShift clusters. 
+A profile is an identifier that the Cluster Version Operator uses to determine
+which manifests to apply. Operators can be excluded completely or can have different
+manifests for each supported profile.
+
+## Motivation
+
+In order to support different deployment models in which not all operators rendered by
+the CVO by default are needed, a mechanism is needed by which we tell the CVO which 
+manifests to include or exclude. One such deployment is the hosted control plane case
+in which the control plane components and corresponding operators should not be rendered
+by the CVO.
+
+
+### Goals
+
+- Equip the CVO to include/exclude manifests based on a matching identifier (profile)
+
+### Non-Goals
+
+- Define which manifests should be excluded/included for each use case
```

Comment:
The feedback I received on the idea of using cluster profiles for bare metal reiterated something quite clearly - the goal here is to address the hypershift case initially, and then the CRC case, and isn't a generic mechanism that we're keen to solve other problems with. I think that would be good to capture more clearly in the motivation/goals/non-goals section.

Example of that feedback - https://github.com/openshift/enhancements/pull/212#issuecomment-587496071

Reply:
Ack, adding that to the motivation

Reply:
Thanks, looks good

Reply:
(can be resolved)